### ISSUE TITLE
NPC: Rokaro

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#66]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Desolace / Shadowprey Village / NPC: Rokaro 
Link: https://www.wowhead.com/wotlk/npc=10182/rokaro

Greeting: Rokaro replaced Rexxar as Champion of the Horde after the release of TBC with Rexxar moving to Outland. Depending on the expansion the player have installed, this NPC's ID should play back different greeting text. With WotLK installed, is currently playing Rexxar's greeting.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_TBC_v1.4.
